tests/test_config.py

```
@@ -1,218 +1,152 @@
-# Copyright (c) 2016-2022 Knuth Project developers.
+# Copyright (c) 2016-present Knuth Project developers.
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 import kth_native
 
+
 def hash_to_hex(hash):
```

Ruff (0.15.9)

[error] Function argument `hash` is shadowing a Python builtin

(A002)